tests/test_pretrial_verified_law_rescue.py

```
@@ -0,0 +1,315 @@
+
+
+def test_pretrial_kazakh_rescue_releases_verified_document() -> None:
+    async def scenario() -> None:
+        install_kazakh_legal_bridge()
+        install_kazakh_article_forms()
+        research = _verified_research_kk()
+        initial = _draft_with_bad_article_kk()
+        assert pretrial_quality_issues(initial, research)
+        calls: list[dict] = []
+
+        async def original(_self, _context, _research, language="kk"):
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `original`

(ANN202)